doc/api/crypto.md

```diff
@@ -2057,6 +2081,68 @@ Based on the recommendations of [NIST SP 800-131A][]:
 
 See the reference for other recommendations and details.
 
+### CCM mode
+
+CCM is the second supported AEAD algorithm. Applications which use this mode
```

Comment:
"the second"? can it just be "one of the two". I think also a link to a reference for AEAD might be helpful, there's going to be very few people that have a clue what it is. Maybe Wikipedia would be enough.